src/main/java/com/cozyhome/onlineshop/config/WebSecurityConfig.java

```diff
@@ -62,12 +65,18 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
 		http.csrf(csrf -> csrf.disable())
 				.exceptionHandling(exception -> exception.authenticationEntryPoint(unauthorizedHandler))
 				.sessionManagement(session -> session.sessionCreationPolicy(SessionCreationPolicy.STATELESS))
-				.authorizeHttpRequests(auth -> auth.requestMatchers(AUTH_WHITELIST).permitAll()
-						.anyRequest().authenticated());
+				.authorizeHttpRequests(
+						auth -> auth.requestMatchers(AUTH_WHITELIST).permitAll().anyRequest().authenticated());
```

Comment:
It needs to go back to the way it was. ".anyRequest().authenticated()" is spelled on a new line.

Reply:
Do we really need comments for formatting things?